magic/rotation.py

```diff
@@ -216,18 +221,26 @@ def read_rotation_files() -> Tuple[int, int, List[Card]]:
         c = process_score(name, hits, cs, runs, latest_list)
         if c is not None:
             cards.append(c)
+    redis.store('decksite:rotation:summary:runs', runs)
+    redis.store('decksite:rotation:summary:runs_percent', runs_percent)
+    redis.store('decksite:rotation:summary:cards', cards)
     return (runs, runs_percent, cards)
```

Comment:
When/how does this expire? We need it to pick up the new file as soon as one comes into existence during rotation.

Reply:
I tried to make `rotation_script` responsible for clearing: https://github.com/PennyDreadfulMTG/Penny-Dreadful-Tools/pull/6688/files#diff-5d117356d255d8dcb81cd6bcb5a39820R34